src/components/AccommodationDetails/AccommodationDetails.module.css

```diff
@@ -0,0 +1,64 @@
+.container {
+  padding: 60px;
+}
+
+.image {
+  width: 100%;
+  border-radius: 20px;
+  height: 427px;
+  object-fit: cover;
+  margin: 10px;
+}
+
+.title {
+  font-weight: 400;
+  font-size: 34px;
+  line-height: 120%;
+  letter-spacing: 0.25px;
+  color: #000000;
+}
+
+.inline {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 20px;
+}
+
+.subtitle {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 150%;
+  letter-spacing: 0.15px;
+  color: #7c7c7c;
+  margin-top:12px;
+}
+
+.content {
+  margin: 15px;
+  width: 810px;
+  position: relative;
+  float:left;
```

Comment:
I suggest avoiding floats at all costs. The float property was around times of IE when it was used for simple layouts. What it does is it places the content to one side and places the rest of the content around it, which by the sound of it sounds unpredictable. You don't know how the rest of the layout will be. You should use the modern alternatives. In this case `flex`. There is also `grid`.
There is no credible source for what i said but you can google around a bit and find similar answers to what i've said.